Bio/SearchIO/BlastIO/blast_xml.py

```diff
@@ -574,19 +572,17 @@ def __iter__(self):
         """Iterate over BlastXmlIndexer yields qstart_id, start_offset, block's length."""
         qstart_mark = self.qstart_mark
         qend_mark = self.qend_mark
-        blast_id_mark = _as_bytes("Query_")
+        blast_id_mark = b"Query_"
         block_size = self.block_size
         handle = self._handle
         handle.seek(0)
         re_desc = re.compile(
-            _as_bytes(
-                r"<Iteration_query-ID>(.*?)"
-                r"</Iteration_query-ID>\s+?"
-                "<Iteration_query-def>"
-                "(.*?)</Iteration_query-def>"
-            )
+            br"<Iteration_query-ID>(.*?)"
+            br"</Iteration_query-ID>\s+?"
```

Comment:
`b"</Iteration_query-ID>\s+?"`

Reply:
No we need the raw mode otherwise ``\s`` triggers a warning:

```
W605 invalid escape sequence '\s'
```

See e.g. https://github.com/PyCQA/pycodestyle/issues/755 and links.

Reply:
Looks like it will raise a non-silent warning in Python 3.9
https://bugs.python.org/issue32912